lib/Conversion/TorchToLinalg/TorchToLinalg.cpp

```diff
@@ -2584,6 +2584,93 @@ struct ConvertReductionOp : ConversionPattern {
 };
 } // namespace
 
+// Computes maxpool2d for AtenMaxPool2dOp and AtenMaxPool2dWithIndicesOp.
+template <typename OpTy>
+static LogicalResult
+computeMaxPool2d(OpTy op, Location loc, ConversionPatternRewriter &rewriter,
+                 Value self, Value ceilMode, Value &result) {
```

Comment:
nit: `loc` can be computed from `op`, so it does not need to be an argument. Also, if we pattern-match on `ceilMode`, then we can use the `ceilMode = op.ceil_mode()`, which means we don't need to have that argument either.